pkg/event/event.go

```diff
@@ -44,39 +44,39 @@ var m = map[string]string{
 // New create new KubewatchEvent
 func New(obj interface{}, action string) Event {
 	var namespace, kind, component, host, reason, status, name string
-	if apiService, ok := obj.(*api.Service); ok {
+	if apiService, ok := obj.(*v1.Service); ok {
```

Comment:
Thanks Gus. That duplicated codes have been there for a year since I had limited knowledge about k8s. Will update it!